server/vulnerabilities/nvd/cve.go

```diff
@@ -131,16 +131,35 @@ func TranslateCPEToCVE(
 		return nil, nil
 	}
 
-	var vulns []fleet.SoftwareVulnerability
+	vulns := make(map[string]fleet.SoftwareVulnerability)
 	for _, file := range files {
-		r, err := checkCVEs(ctx, ds, logger, parsed, file, collectVulns)
+		foundVulns, err := checkCVEs(ctx, ds, logger, parsed, file, collectVulns)
```

Comment:
can you add a comment explaining why we are using a map here? I assume it's to remove duplicate vulnerabilities, but that's not obvious because I wouldn't expected vulnerabilities to be the same across multiple files.